### DISCUSSION TITLE
Find cell-type-specific peak in two biological sample

### DISCUSSION THREAD

**Discussion by Mengzhiyaa** [GreenleafLab/ArchR#1007] [Questions / Documentation]
Hi, I'm analysis scATAC-seq generated from different biological conditions using ArchR, and I perform the cluster-specific motif enrichment analysis. And I want to perform motif enrichment of the different accessibility peaks of same clusters under differential conditions. Will you help me with this problem.

Thanks.

**Comment by rcorces** [comment]
If I understand your question, this is covered in the manual or in the function documentation. Please review those carefully
